src/benchmark/file_based_benchmark.cpp

```diff
@@ -23,7 +23,7 @@ int main(int argc, char* argv[]) {
 
   // clang-format off
   cli_options.add_options()
-      ("table_path", "Directory containing the Tables", cxxopts::value<std::string>()->default_value("")) // NOLINT
+      ("table_path", "Directory containing the Tables as csv, tbl or binary files", cxxopts::value<std::string>()->default_value("")) // NOLINT
```

Comment:
CSVs require meta files, don't they?

Reply:
Yes. I will try to document them as thoroughly as possible in the one line that they get in `--help`.